### ISSUE TITLE
snap update 23 to 24 loses shared folders

### ISSUE THREAD

**Issue by merkleID** [nextcloud-snap/nextcloud-snap#2103]
#### Describe the bug
snap updating from 23-latest to 24 results in a complete loss of data

this is from journalctl:

Jun 13 11:06:35 mtradenc nextcloud.apache[6782]: Making sure nextcloud is setup...
Jun 13 11:06:35 mtradenc nextcloud.apache[6938]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:35 mtradenc nextcloud.apache[6938]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:35 mtradenc nextcloud.apache[6938]: System config value redis => host set to string /tmp/sockets/redis.sock
Jun 13 11:06:35 mtradenc nextcloud.apache[7009]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:35 mtradenc nextcloud.apache[7009]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:35 mtradenc nextcloud.apache[7009]: System config value redis => port set to integer 0
Jun 13 11:06:35 mtradenc nextcloud.apache[7080]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:35 mtradenc nextcloud.apache[7080]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:35 mtradenc nextcloud.apache[7080]: System config value memcache.locking set to string \OC\Memcache\Redis
Jun 13 11:06:35 mtradenc nextcloud.apache[7151]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:35 mtradenc nextcloud.apache[7151]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:36 mtradenc nextcloud.apache[7151]: System config value memcache.local set to string \OC\Memcache\Redis
Jun 13 11:06:36 mtradenc nextcloud.apache[7222]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:36 mtradenc nextcloud.apache[7222]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:36 mtradenc nextcloud.apache[7222]: No such app enabled: updatenotification
Jun 13 11:06:36 mtradenc nextcloud.apache[7294]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:36 mtradenc nextcloud.apache[7294]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:36 mtradenc nextcloud.apache[6825]: Making sure nextcloud is fully upgraded...
Jun 13 11:06:36 mtradenc nextcloud.apache[7366]: Nextcloud or one of the apps require upgrade - only a limited number of commands are available
Jun 13 11:06:36 mtradenc nextcloud.apache[7366]: You may use your browser or the occ upgrade command to do the upgrade
Jun 13 11:06:36 mtradenc nextcloud.apache[7366]: Setting log level to debug
Jun 13 11:06:36 mtradenc nextcloud.apache[7366]: Turned on maintenance mode
Jun 13 11:06:36 mtradenc nextcloud.apache[7366]: Updating database schema
Jun 13 11:06:36 mtradenc nextcloud.apache[7366]: Updated database
Jun 13 11:06:38 mtradenc nextcloud.apache[7366]: Starting code integrity check...
Jun 13 11:06:51 mtradenc nextcloud.apache[7366]: Finished code integrity check
Jun 13 11:06:51 mtradenc nextcloud.apache[7366]: Update successful
Jun 13 11:06:51 mtradenc nextcloud.apache[7366]: Turned off maintenance mode
Jun 13 11:06:51 mtradenc nextcloud.apache[7366]: Resetting log level
Jun 13 11:06:51 mtradenc nextcloud.apache[6782]: All set! Running httpd...
Jun 13 11:06:51 mtradenc nextcloud.apache[7413]: Certificates have been activated: using HTTPS only
Jun 13 11:06:51 mtradenc nextcloud.apache[7413]: Certificates appear self-signed: disabling HSTS
Jun 13 11:06:51 mtradenc nextcloud.apache[7413]: HTTP compression is disabled
Jun 13 11:06:52 mtradenc nextcloud.apache[7469]: AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 127.0.1.1. Set the 'ServerName' directive globally to suppress this message
Jun 13 11:06:52 mtradenc nextcloud.nextcloud-fixer[5066]: done
Jun 13 11:06:52 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/1_upgrade.sh
Jun 13 11:06:52 mtradenc nextcloud.nextcloud-fixer[7693]: Nextcloud is already latest version
Jun 13 11:06:52 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/2_update-apps.sh
Jun 13 11:06:53 mtradenc nextcloud.nextcloud-fixer[7765]: groupfolders new version available: 12.0.0
Jun 13 11:06:53 mtradenc nextcloud.nextcloud-fixer[7765]: groupfolders updated
Jun 13 11:06:53 mtradenc nextcloud.nextcloud-fixer[7765]: user_usage_report new version available: 1.8.0
Jun 13 11:06:53 mtradenc nextcloud.nextcloud-fixer[7765]: user_usage_report updated
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7865]: Nextcloud is already latest version
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/3_add-missing-indices.sh
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the share table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the filecache table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the twofactor_providers table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the login_flow_v2 table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the whats_new table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the cards table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the cards_properties table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the calendarobjects_props table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the schedulingobjects table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the oc_properties table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the oc_jobs table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Check indices of the oc_direct_edit table.
Jun 13 11:06:54 mtradenc nextcloud.nextcloud-fixer[7936]: Adding direct_edit_timestamp index to the oc_direct_edit table, this can take some time...
Jun 13 11:06:55 mtradenc nextcloud.nextcloud-fixer[7936]: oc_direct_edit table updated successfully.
Jun 13 11:06:55 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/4_add-missing-columns.sh
Jun 13 11:06:55 mtradenc nextcloud.nextcloud-fixer[8011]: Check columns of the comments table.
Jun 13 11:06:55 mtradenc nextcloud.nextcloud-fixer[8011]: Done.
Jun 13 11:06:55 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/5_regenerate-assets.sh
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Repair MySQL collation
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - All tables already have the correct collation -> nothing to do
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Repair mime types
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clean tags and favorites
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 tags of deleted users have been removed.
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 tags for delete files have been removed.
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 tag entries for deleted tags have been removed.
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 tags with no entries have been removed.
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Repair invalid shares
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Move .step file of updater to backup location
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add move avatar background job
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - Repair step already executed
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add preview cleanup background jobs
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:  - Migrate oauth2_clients table to nextcloud schema
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - Update the oauth2_access_tokens table schema.
Jun 13 11:06:56 mtradenc nextcloud.nextcloud-fixer[8083]:      - Update the oauth2_clients table schema.
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Fix potential broken mount points
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - No mounts updated
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Repair language codes
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Install new core bundle components
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add log rotate job
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clear frontend caches
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Image cache cleared
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - SCSS cache cleared
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - JS cache cleared
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clear every generated avatar on major updates
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add preview background cleanup job
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Queue a one-time job to cleanup old backups of the updater
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Cleanup invalid photocache files for carddav
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add background job to cleanup login flow v2 tokens
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Remove potentially over exposing share links
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - No need to remove link shares.
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clear access cache of projects
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Reset generated avatar flag
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Keep legacy encryption enabled
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Check encryption key format
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Remove old dashboard app config data
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add job to cleanup the bruteforce entries
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Queue a one-time job to check for user uploaded certificates
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Repair DAV shares
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add background job to set the lookup server share state for users
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Update hook for Appointments app
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - appointments UpdateHook finished
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Update name of the stored view
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Upgrading Circles App
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Fix component of birthday calendars
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 11 birthday calendars updated.
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Regenerating birthday calendars to use new icons and fix old birthday events without year
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Repair step already executed
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Fix broken values of calendar objects
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:     0 [>---------------------------]
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Registering building of calendar search index as background job
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Repair step already executed
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Register building of social profile search index as background job
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Registering background jobs to update cache for webcal calendars
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Added 0 background jobs to update webcal calendars
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Registering building of calendar reminder index as background job
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Repair step already executed
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clean up orphan event and contact data
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 events without a calendar have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 properties without an events have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 changes without a calendar have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 cached events without a calendar subscription have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 changes without a calendar subscription have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 contacts without an addressbook have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 properties without a contact have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 changes without an addressbook have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Remove activity entries of private events
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Removed 0 activity entries
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clean up old calendar subscriptions from deleted users that were not cleaned-up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:     0 [->--------------------------]
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 calendar subscriptions without an user have been cleaned up
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Fix the share type of guest shares when migrating from ownCloud
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Copy the share password into the dedicated column
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Set existing shares as accepted
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Restore default tags that are missing
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Looking up default tags
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 2 accounts to check found
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  0/2 [>---------------------------]   0%
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  2/2 [============================] 100%
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add a generated message-id to all Mail messages that have none
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Looking up messages without a message-id
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Purify and migrate collected mail addresses
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:     0 [------->--------------------]
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Insert background jobs for all accounts
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  0/2 [>---------------------------]   0%
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  2/2 [============================] 100%
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Make Mail itinerary extractor executable
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Create or update provisioned Mail accounts
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - 0 accounts provisioned
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Repair Broken Threads for all mail accounts
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Repaired threads, 0 messages updated
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Clean up meta table
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Update OAuth token expiration times
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Switches from default updater server to the customer one if a valid subscription is available
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:      - Repair step already executed
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Send an admin notification if monthly report is disabled
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Add background job to check for backup codes
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8083]:  - Populating added database structures for workflows
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/6_add-missing-primary-keys.sh
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8156]: Check primary keys.
Jun 13 11:06:57 mtradenc nextcloud.nextcloud-fixer[8156]: Done.
Jun 13 11:06:58 mtradenc nextcloud.nextcloud-fixer[7628]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/99_notify_admins.sh
Jun 13 11:06:58 mtradenc nextcloud.nextcloud-fixer[8163]: Notifying admins of update from 23.0.5snap1 to 24.0.1snap1...
Jun 13 11:07:12 mtradenc nextcloud.nextcloud-fixer[8163]: done
Jun 13 11:07:12 mtradenc nextcloud.nextcloud-fixer[5066]: Enabling maintenance mode...
Jun 13 11:07:12 mtradenc nextcloud.nextcloud-fixer[5066]: done
Jun 13 11:07:15 mtradenc nextcloud.nextcloud-fixer[11978]: run-parts: executing /snap/nextcloud/31033/fixes/existing-install/maintenance/1_convert-filecache-bigint.sh
Jun 13 11:07:15 mtradenc nextcloud.nextcloud-fixer[12044]: Nextcloud is in maintenance mode, hence the database isn't accessible.
Jun 13 11:07:15 mtradenc nextcloud.nextcloud-fixer[12044]: Cannot perform any command except 'maintenance:mode --off'
Jun 13 11:07:16 mtradenc nextcloud.nextcloud-fixer[12044]: All tables already up to date!
Jun 13 11:07:16 mtradenc nextcloud.nextcloud-fixer[5066]: Disabling maintenance mode...
Jun 13 11:07:16 mtradenc nextcloud.nextcloud-fixer[5066]: done
Jun 13 11:07:19 mtradenc systemd[1]: snap.nextcloud.nextcloud-fixer.service: Succeeded.
-- Subject: Unit succeeded
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support

simply put all users do not see anymore folders belonging to group.
As my customer called me this morning because they couldn't find their files anymore, I investigated and then decided to restore to saturday morning backup.

As I restored (version 23-latest) everything seemed to be back in place, but after a few minutes thing went back to disaster.
I explored journalctl, and found this happening: after this, all group folders are no more accessible/visible.

**Comment by scubamuc** [comment]
can you confirm this is happening to "shared folders"? 
this seems to be an issue with "groupfolders"!

**snap update 23 to 24 loses groupfolders** 
(https://github.com/nextcloud/groupfolders/issues/1990)

the suggested upgrade to "groupfolders 12.0" is not available yet.

my workaround for the time being: revert to 23.0.5snap1 and wait for "groupfolders 12.0"
![grafik](https://user-images.githubusercontent.com/54933878/174750499-ad78b373-fc94-4f03-a933-36b06e5ea5f1.png)

**Comment by scubamuc** [comment]
@merkleID could you please change the issue heading to:

**snap update 23 to 24 loses groupfolders**

**Comment by scubamuc** [comment]
**snap update 23 to 24 loses groupfolders... resolved**

- snap update 23.0.5 to 24.0.1 
- find groupfolders in deactivated apps
- activate groupfolders (version 12.0)
- groupfolders are visible again
- update success

**Comment by merkleID** [comment]
> can you confirm this is happening to "shared folders"? this seems to be an issue with "groupfolders"!

you're right, it was groupfolders and not shared folders

**Comment by merkleID** [comment]
> **snap update 23 to 24 loses groupfolders... resolved**
> 
>     * snap update 23.0.5 to 24.0.1
> 
>     * find groupfolders in deactivated apps
> 
>     * activate groupfolders (version 12.0)
> 
>     * groupfolders are visible again
> 
>     * update success

I blocked updates too, now I will try your suggestion

**Comment by pachulo** [comment]
This looks like a groupfolders upstream bug.
